### ISSUE TITLE
Avisynth plugin build probably breaks again

### ISSUE THREAD

**Issue by MaverickTse** [MaverickTse/lsw-build2#18]
As a result of this commit:
https://github.com/VFR-maniac/L-SMASH-Works/commit/71859e2428c1d8cb7ed44dd2121ed95be9a8a233

Probably the AviSynth build will break. I haven't test it yet, but Mingw libs have extension *.a.

The VC linker will probably complains on missing libs or external references

I don't think I'll bother to fix this again... just a pain to mix MSYS2, Mingw and MSVC...

**Comment by MaverickTse** [comment]
There are several ways to tackle this:
1. make a patch to revert LSW commit 71859e2 before building
2. build everything upstream with MSVC
3. rename .a to .lib (not recommended, since that breaks pkgconfig and cause problem to other software)

**Comment by MaverickTse** [comment]
Updates:
According to Takuan, it is not a good idea to mix libs built with Mingw and MSVC in a static build. That is probably the "problem" cited in that commit. (Personally I didn't ever encounter any problem...)

So I guess it is safe to build everything static with MSVC only, or build shared using any compiler of your choice.

**Comment by yumetodo** [comment]
ffmpegもL-SMASHもC結合だったと思うんですが、なんで「混ぜるな危険」なんだろう・・・。

Avisynth向けのビルドバッチの除去を提案します。
すべてMSVCでcompileしないといけないのならば、ffmpegやL-SMASHをgccでビルドするこのscript群とは、相容れないものに思えるからです(batとslnでやればいい)

**Comment by MaverickTse** [comment]
I agree. But don't delete the files yet, just comment out the calls to bld_lsw_avs(64).sh so that those scripts will not run by default.

**Comment by MaverickTse** [comment]
Added a WIKI entry on how to build Avisynth plugin
